src/main/java/org/terasology/launcher/game/GameStarter.java

```diff
@@ -49,8 +52,19 @@
         processParameters.add("-DlogOverrideLevel=" + logLevel.name());
         processParameters.addAll(javaParams);
 
+        // Locate the main game jar. Currently, Terasology has custom build logic to put libraries into a 'libs'
+        // (plural) subdirectory. As we plan to switch to using default Gradle behavior we have to do a quick check
+        // how the game distribution was build (i.e., custom 'libs' or default 'lib').
+        //TODO: this should probably be part of ReleaseMetadata and be determined further up the hierarchy
```

Comment:
I'm okay with it being here. Unless/until we start having different names or locations of jar files on a regular basis.